src/lib/orion_websocket/ws.cpp

```diff
@@ -176,6 +182,21 @@ void *runWS(void *ptr)
     pthread_mutex_lock(&mtx);
     lws_service(ws->ctx, 50);
     pthread_mutex_unlock(&mtx);
+/*
```

Comment:
Oh! Sorry my mistake. This file should be out of this commit. It was fixed as soon as I can put my hands in the laptop.